### PR TITLE
Add feedback vibration using Taptic Engine on iOS

### PR DESCRIPTION
**Pull request by ichan-mb**
On iOS, now you can do feedback vibration by leveraging Taptic Engine. There are 9 types of vibration :
- Soft
- Rigid
- light
- Medium
- Heavy
- Success
- Warning
- Error
- Selection

To activate it, just pass those value to `VibrationType` property
 
**Example**
 ```
<StackPanel Margin="20">
	<!-- On iOS it will do a 'heavy' vibration using Taptic Engine -->
	<!-- On Android it will use standard vibration for a duration 0.4 second  -->
	<Button Margin="10" Text="Heavy Vibrate">
		<Clicked>
			<Vibrate VibrationType="Heavy" />
		</Clicked>
	</Button>
</StackPanel>
```

This PR contains:
- [x] Changelog
- [x] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Vibration/Vibration.uno [hunk 1]
```diff
@@ -25,6 +39,168 @@ namespace Fuse.Vibration
...
+			AudioServicesPlayAlertSound(kSystemSoundID_Vibrate);
+		#endif
+		@}
+
+		[Foreign(Language.ObjC)]
+		static extern(iOS) void PerformMedium()
+		@{
+		#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+			dispatch_async(dispatch_get_main_queue(), ^{
+				UIImpactFeedbackGenerator * feedback = [[UIImpactFeedbackGenerator alloc] initWithStyle:UIImpactFeedbackStyleLight];
```
Comment (mortend) [inline Source/Fuse.Vibration/Vibration.uno]: This code looks identical to `PerformLight()`, do you mean `initWithStyle:UIImpactFeedbackStyleMedium` ?

File: Source/Fuse.Vibration/Vibration.uno [hunk 2]
```diff
@@ -25,6 +39,168 @@ namespace Fuse.Vibration
...
+			AudioServicesPlayAlertSound(kSystemSoundID_Vibrate);
+		#endif
+		@}
+
+		[Foreign(Language.ObjC)]
+		static extern(iOS) void PerformHeavy()
+		@{
+		#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+			dispatch_async(dispatch_get_main_queue(), ^{
+				UIImpactFeedbackGenerator * feedback = [[UIImpactFeedbackGenerator alloc] initWithStyle:UIImpactFeedbackStyleLight];
```
Comment (mortend) [inline Source/Fuse.Vibration/Vibration.uno]: This code looks identical to `PerformLight()`, do you mean `initWithStyle:UIImpactFeedbackStyleHeavy` ?

File: Source/Fuse.Vibration/Vibrate.uno [hunk 3]
```diff
@@ -17,22 +17,61 @@ namespace Fuse.Vibration
...
 					<Clicked>
 						<Vibrate Duration="5" />
 					</Clicked>
 				</Button>
 			</StackPanel>
+
+		On iOS you can do vibration by leveraging Taptic Engine. There are 9 types of vibration :
+		* Soft
+		* Rigid
+		* light
```
Comment (mortend) [inline Source/Fuse.Vibration/Vibrate.uno]: `Light` should probably start with an upper-case letter, same as the other bullet points.

File: Source/Fuse.Vibration/Vibration.uno (hunk 1 above)
Comment (ichan-mb) [reply Source/Fuse.Vibration/Vibration.uno]: Ops! sorry about that, the evil of copy and paste, fix on the way

File: Source/Fuse.Vibration/Vibration.uno (hunk 2 above)
Comment (ichan-mb) [reply Source/Fuse.Vibration/Vibration.uno]: the same evil of copy and paste, fix on the way

**Pull request merged**